opentelemetry-otlp/examples/basic-otlp/src/main.rs

```diff
@@ -86,6 +124,8 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
         );
         span.set_attribute(ANOTHER_KEY.string("yes"));
 
+        error!(target: "my-target", "hello from {}. My price is {}. I am also inside a Span!", "banana", 2.99);
```

Comment:
sorry for nitpicking,- It's not really an error message. Can be made as `info!` after changing the max level, or else fine to keep it as such.